app/helpers/place/finish_line_helper.rb

```diff
@@ -0,0 +1,29 @@
+module Place
+  module FinishLineHelper
+    MAPS_API_KEY = Rails.application.secrets.maps_api_key
+    HOST = 'https://maps.googleapis.com/maps/api/staticmap?'.freeze
+
+    def finish_line_static_map_path(finish_line)
+      center = finish_line.center
+      params = {
+        size: '600x400',
+        center: [center[:latitude], center[:longitude]].join('+'),
+        zoom: 15,
+        maptype: 'satellite',
+        path: polyline_params(finish_line),
+        key: MAPS_API_KEY
+      }
+      params_str = params.reduce('') { |str, value| str + '&' + value.join('=') }
+      HOST + params_str
+    end
+
+    private
+
+    def polyline_params(finish_line)
```

Comment:
It's not really a private, but globally accessible. So I'd advise to rename it to `finish_line_static_polyline` and remove `private` keyword above.